.gitignore

```diff
@@ -7,6 +7,8 @@ test/deps/devlxd-client
 test/macaroon-identity/macaroon-identity
 *~
 tags
+manifest
+manifest.uuid
```

Comment:
This is a Makefile bug, those files should never be there, please drop this commit.

Reply:
@stgraber have done thanks